Zed kolla build config

Alex-Welsh opened the pull request:
After this PR, Zed kolla container images should all be buildable and AIO CI should pass (the branch protection rules do still need a bit of work though)

Alex-Welsh committed:
Fix kolla image builds for new name scheme

Alex-Welsh committed:
Fix keystone kolla image build

This fixes keystone image build failures due the pinned version
of liblasso being incorrect for ubuntu jammy

Alex-Welsh committed:
Fix prometheus cadvisor kolla image build

Removed the Prometheus cadvisor version override to use kolla default,
which is now more up-to-date than the previously used version. This
fixes the kolla image build.

Alex-Welsh committed:
Fix prometheus node exporter kolla image build

This change updates the node exporter sha256 hash to match the newer
(pinned) version we are using. This was not required in the yoga cycle
as the hash was not checked.

Alex-Welsh committed:
Update kolla/globals.yml for Zed release

Alex-Welsh committed:
Fix lint and comments

Alex-Welsh committed:
Use Rocky Linux 9 as base for kayobe-automation

By default, kayobe-automation uses CentOS Stream 8 as the base image
for its kayobe docker image. As it doesn't support python 3.8 (a
requirement for the Zed release), it must be overridden to use Rocky
Linux 9.

Alex-Welsh committed:
Disable selinux in CI AIO env

Changes to the selinux role now mean that host configure will fail if
the current selinux state does not match the desired state, and
selinux_do_reboot is disabled.

Alex-Welsh commented:
AIOs should pass, but it's pulling the wrong version of kayobe-automation https://github.com/stackhpc/stackhpc-kayobe-config/actions/runs/6014619558/job/16314730497?pr=606#step:2:482 I'll see if I can get it fixed tomorrow morning

Alex-Welsh committed:
Update tox CI to use python 3.10

Alex-Welsh committed:
Update kayobe-automation submodule

markgoddard commented on etc/kayobe/kolla/globals.yml:
```diff
@@ -6,63 +6,14 @@
-kolla_base_distro: "{% raw %}{{ 'centos' if ansible_facts.distribution == 'Rocky' and ansible_facts.distribution_major_version == '8' else ansible_facts.distribution | lower }}{% endraw %}"
+kolla_base_distro: "{% raw %}{{ ansible_facts.distribution | lower }}{% endraw %}"
 
 kayobe_image_tags:
```
I don't see this referenced anywhere.

Alex-Welsh replied:
It will be necessary again as soon as we update an individual tag, but probably not needed now

markgoddard replied:
I thought we were going to use `openstack_tag` in globals.yml to set the default tag?

Alex-Welsh committed:
Override openstack_tag

Now overriding openstack_tag directly rather than indirectly via
kolla_openstack_release.
